cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsClient.java

```diff
@@ -323,8 +323,4 @@ private AwsSessionCredentialProvider createAwsSessionCredentialProvider(AwsCrede
         proxyFactory.addAspect(new AmazonClientExceptionHandler(awsCredentialView, region, sdkClientExceptionMapper));
         return proxyFactory.getProxy();
     }
-
-    public void setAwsSessionCredentialClient(AwsSessionCredentialClient awsSessionCredentialClient) {
```

Comment:
If I remember correctly it was there to enable setter bean injection, did something change that it is no longer required?

Reply:
@akanto removed the bean injection in this PR: https://github.com/hortonworks/cloudbreak/commit/2436b25fd0abd36a201cf92e35e73513f658ffc9